circuits/cpp/src/aztec3/circuits/kernel/private/native_private_kernel_circuit_inner.cpp

```diff
@@ -137,7 +137,7 @@ KernelCircuitPublicInputs<NT> native_private_kernel_circuit_inner(DummyBuilder&
 
     // TODO(dbanks12): may need to comment out hash check in here according to TODO above
     // TODO(jeanmon) FIXME - https://github.com/AztecProtocol/aztec-packages/issues/671
-    // common_validate_call_stack(builder, private_inputs.private_call);
+    common_validate_call_stack(builder, private_inputs.private_call);
```

Comment:
I guess we can remove the `TODO` comments above this line, if this works now? :)